fdm-app/app/routes/app.addfarm.$b_id_farm.cultivations.tsx

```
@@ -125,4 +126,41 @@ export default function Index() {
-}
\ No newline at end of file
+}
+
+interface SidebarNavProps extends React.HTMLAttributes<HTMLElement> {
+    items: {
+        href: string
+        title: string
+    }[]
+}
+
+export function SidebarNav({ className, items, ...props }: SidebarNavProps) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'SidebarNav'. Consider to delete it or rename it.

'SidebarNav' is defined here:


(lint/suspicious/noRedeclare)